kubernetes/skaffold/src/main/kotlin/com/google/kubernetes/tools/skaffold/SkaffoldExecutorService.kt

```diff
@@ -160,10 +160,11 @@ data class SkaffoldExecutorSettings(
     val defaultImageRepo: String? = null
 ) {
 
-    /** Execution mode for Skaffold, single run, continuous development, etc. */
+    /** Execution mode for Skaffold, single run, continuous development (run or debug mode), etc. */
     enum class ExecutionMode(val modeFlag: String) {
         SINGLE_RUN("run"),
-        DEV("dev")
+        DEV("dev"),
+        DEBUG("debug")
```

Comment:
note: this command obviously won't work yet (unless you built skaffold manually); but once its merged in, then things should work as expected.

Reply:
Should we hide this behind the feature flag until it's ready to avoid errors? Or we are just going to wait with merging this PR until it's done?

Reply:
I think its ok to keep this for now. I've commented out the program runner in `skaffold.xml` so this execution mode can never be consumed (the debug button will not be available, as it currently behaves).